src/components/IncidentInfo/IncidentInfo.js

```diff
@@ -21,20 +23,29 @@ const IncidentInfo = ({ id }) => {
   //   }
   // }
 
+  useEffect(() => {
+    dispatch(getIncidentInfo(id))
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [dispatch])
```

Comment:
I'm surprised this threw Travis CI an error when it didn't locally.  Glad this command could help.